zephyr/include/rtos/string.h

```diff
@@ -11,6 +11,9 @@
 #include <stddef.h>
```

Comment:
This patch looks good. We usually add a prefix to commit subject. 

Something like this: `zephyr: rtos`. 

Also, please imperative mood in the subject line:


e.g `zephyr: rtos: Allow memory operations to be called from cpp code`

See: https://cbea.ms/git-commit/#imperative

Reply:
fixed